flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/assigners/ChunkSplitter.java

```diff
@@ -224,21 +235,73 @@ private MySqlSnapshotSplit createSnapshotSplit(
     // ------------------------------------------------------------------------------------------
 
     /** Checks whether split column is evenly distributed across its range. */
-    private static boolean splitColumnEvenlyDistributed(Column splitColumn) {
-        // only column is auto-incremental are recognized as evenly distributed.
-        // TODO: we may use MAX,MIN,COUNT to calculate the distribution in the future.
-        if (splitColumn.isAutoIncremented()) {
-            DataType flinkType = MySqlTypeUtils.fromDbzColumn(splitColumn);
-            LogicalTypeRoot typeRoot = flinkType.getLogicalType().getTypeRoot();
-            // currently, we only support split column with type BIGINT, INT, DECIMAL
-            return typeRoot == LogicalTypeRoot.BIGINT
-                    || typeRoot == LogicalTypeRoot.INTEGER
-                    || typeRoot == LogicalTypeRoot.DECIMAL;
+    private static boolean splitColumnEvenlyDistributed(
```

Comment:
Would be better to call `isSplitColumnEvenlyDistributed`, otherwise, this methods sounds like a verb.